### PR TITLE
Do not show `False` conditions except for Ready

### PR DESCRIPTION
**Pull request by lukaszo**
Fixes: #25773

### CODE REVIEW

**Comment by derekwaynecarr** [comment]
I don't necessarily agree with this PR change.  It's useful to know when a condition transitioned from true to false.

**Comment by derekwaynecarr** [comment]
See: https://github.com/kubernetes/kubernetes/issues/25773#issuecomment-220339386

Please do not merge this until we get consensus.

**Comment by lukaszo** [comment]
I may add a flag to kubectl to show all conditions. Will it be enough for you?

File: pkg/kubectl/cmd/describe.go [hunk 1]
```diff
@@ -95,6 +95,7 @@ func NewCmdDescribe(f *cmdutil.Factory, out io.Writer) *cobra.Command {
 	cmdutil.AddRecursiveFlag(cmd, &options.Recursive)
 	cmd.Flags().StringP("selector", "l", "", "Selector (label query) to filter on")
 	cmd.Flags().BoolVar(&describerSettings.ShowEvents, "show-events", true, "If true, display events related to the described object.")
+	cmd.Flags().BoolVar(&describerSettings.ShowAll, "show-all", false, "If true, display all hidden fields.")
```
Comment (jlowdermilk) [inline pkg/kubectl/cmd/describe.go]: I don't think this flag adds much value; it only applies to nodes, and the help text is too vague to be meaningful. Let's just omit the conditions we don't care about; if you want to see all node conditions including false ones, you can still get them by running `kubectl get nodes $nodename -o yaml`.

**Comment by lukaszo** [comment]
Ok, so I'm closing it for now

**Pull request closed without merging**